<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListViewItem</class>
 <widget class="QWidget" name="ListViewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>137</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="txtId">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSpinBox" name="editAddr">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetAddr">
       <property name="text">
        <string>Set Address</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="txtCurStatus">
       <property name="text">
        <string>CurrentStatus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQueryStatus">
       <property name="text">
        <string>Query Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="txtSetStatus">
     <property name="text">
      <string>Set Status</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="txtVersion">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQueryVersion">
       <property name="text">
        <string>Query Version</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
